### PR TITLE
bugfix: distinguish NotFound error with others about image

### PR DESCRIPTION
**Pull request by Starnop**
Signed-off-by: Starnop <starnop@163.com>

### Ⅰ. Describe what this PR did
 distinguish NotFound error with others about image

### Ⅱ. Does this pull request fix one issue?

None.

### CODE REVIEW

**Comment by allencloud** [comment]
Please try to use `failed to do something` when constructing error or log. @Starnop

File: cri/v1alpha1/cri.go [hunk 1]
```diff
@@ -1000,9 +1000,11 @@ func (c *CriManager) ImageStatus(ctx context.Context, r *runtime.ImageStatusRequ
 
 	imageInfo, err := c.ImageMgr.GetImage(ctx, ref.String())
 	if err != nil {
-		// TODO: separate ErrImageNotFound with others.
-		// Now we just return empty if the error occurred.
-		return &runtime.ImageStatusResponse{}, nil
+		if errtypes.IsNotfound(err) {
```
Comment (fuweid) [inline cri/v1alpha1/cri.go]: why return nil error here? could we check the IsNotfound out of this scope?

Comment (Starnop) [reply cri/v1alpha1/cri.go]: The CRI API declares this :
```
ImageStatus returns the status of the image. If the image is not present, returns a response with ImageStatusResponse.Image set to nil.
```

File: cri/v1alpha1/cri_utils.go [hunk 2]
```diff
@@ -18,6 +18,7 @@ import (
 	"github.com/alibaba/pouch/daemon/mgr"
 	"github.com/alibaba/pouch/pkg/utils"
 
+	"github.com/alibaba/pouch/pkg/errtypes"
```
Comment (HusterWan) [inline cri/v1alpha1/cri_utils.go]: pouch own pkg should not put with the third party pakcage

File: cri/v1alpha2/cri_utils.go [hunk 3]
```diff
@@ -18,6 +18,7 @@ import (
 	"github.com/alibaba/pouch/daemon/mgr"
 	"github.com/alibaba/pouch/pkg/utils"
 
+	"github.com/alibaba/pouch/pkg/errtypes"
```
Comment (HusterWan) [inline cri/v1alpha2/cri_utils.go]: pouch own pkg should not put with the third party pakcage

File: cri/v1alpha1/cri_utils.go [hunk 4]
```diff
@@ -822,17 +823,19 @@ func imageToCriImage(image *apitypes.ImageInfo) (*runtime.Image, error) {
...
+	if errtypes.IsNotfound(err) {
+		err = c.ImageMgr.PullImage(ctx, imageRef, nil, bytes.NewBuffer([]byte{}))
+		if err != nil {
+			return fmt.Errorf("failed to pull sandbox image %q: %v", imageRef, err)
+		}
+		return nil
 	}
 
-	return nil
+	return fmt.Errorf("failed to checkReference sandbox image %q: %v", imageRef, err)
```
Comment (fuweid) [inline cri/v1alpha1/cri_utils.go]: I think we should use other word instead of `checkReference`. how about `failed to check sanbox image`?

**Comment by allencloud** [comment]
LGTM until fix some import sequence. @Starnop

**Pull request merged**